templates/config.json.j2

```diff
@@ -1,25 +1,27 @@
 {
-    "masters": [{% for host in groups['mesosmaster'] %}"{{hostvars[host]['ansible_default_ipv4']['address']}}:5050"{% if not loop.last %},{% endif %}{% endfor %}],
```

Comment:
what are the pros of having this as a variable rather than to deduce it from facts about the hosts in a group? without a mesosmaster group the role won't work. without mesos masters the program won't work. as I add masters I reconverge my whole fleet to reflect and identify new masters. never felt obliged to have the zk setting set. I'm open to referencing another group from the inventory that contains the list of hosts that are serving zk. don't think I would go the variable route for that either. what do you think?